Makefile

```diff
@@ -212,7 +212,7 @@ integration: $(MINISHIFT_BINARY)
 	--run-before-feature="$(RUN_BEFORE_FEATURE)" --test-with-specified-shell="$(TEST_WITH_SPECIFIED_SHELL)" --copy-oc-from="$(COPY_OC_FROM)" $(GODOG_OPTS)
 
 .PHONY: integration_all ## Run all integration tests
-integration_all: GODOG_OPTS = --tags=~coolstore
+integration_all: GODOG_OPTS = --tags=~coolstore\&\&~addon-xpaas
```

Comment:
Sam as https://github.com/minishift/minishift/blob/master/Makefile#L221-L222 , we can merge this without testing.